ultraplot/axes/plot.py

```diff
@@ -2709,10 +2709,13 @@ def _parse_cmap(
         norm = _not_none(norm, "div" if "diverging" in trues else "linear")
         # If using a diverging norm, fair=True, and vcenter not set, default to midpoint
         if norm in ("div", "diverging") or "diverging" in trues:
-            fair = norm_kw.get("fair", True)  # defaults to True
+            # When vmin or vmax is given we cannot be sure the colormap is fair
+            is_fair = vmin is None and vmax is None
+            fair = norm_kw.get("fair", is_fair)
```

Comment:
This bit will still override fair if it is set in norm_kw regardless of the value of vmin/vmax. That seems wrong.